### ISSUE TITLE
Syntax error when a multiline value ends with '='

### ISSUE THREAD

**Issue by dmach** [openSUSE/libeconf#200]
This works fine:
```
[section]
multiline_value =
    first line
    second line
```

But if any of these lines ends with `=`:
```
[section]
multiline_value =
    first line=
    second line
```
econf errors out with the "Missing delimiter" SyntaxError (I'm using python bindings).

**Comment by schubi2** [comment]
Well, up to now the parser does not support the Python indentation style. Just for understanding where it is used for ?

**Comment by schubi2** [comment]
Should be available in the next release (0.7)
Please set:
econf_set_opt ("PYTHON_STYLE=1");
